packages/slice-machine/components/DeleteDocumentsDrawer/index.tsx

```diff
@@ -17,35 +17,22 @@ import { useSelector } from "react-redux";
 import { SliceMachineStoreType } from "@src/redux/type";
 import { ModalKeysEnum } from "@src/modules/modal/types";
 import { isModalOpen } from "@src/modules/modal";
+import { selectAllCustomTypes } from "@src/modules/availableCustomTypes";
+import { AssociatedDocumentsCard } from "./AssociatedDocumentsCard";
 import {
-  AssociatedDocumentsCard,
-  CustomtypeDocuments,
-} from "./AssociatedDocumentsCard";
+  getModelId,
+  hasLocal,
+  LocalOrRemoteCustomType,
+} from "@lib/models/common/ModelData";
+import { ToasterType } from "@src/modules/toaster";
 
-export const sortDocumentCards = (
-  documents: Readonly<Array<CustomtypeDocuments>>
-) =>
-  [...documents].sort(
-    (doc1, doc2) => doc2.numberOfDocuments - doc1.numberOfDocuments
-  );
+export function getCTName(ct: LocalOrRemoteCustomType | undefined): string {
+  if (ct === undefined) {
+    return "Could not find Custom Type";
+  }
 
-// TODO: replace with actual API response
-const AssociatedDocuments = [
-  { ctName: "Legals1", numberOfDocuments: 23, link: "https://prismic.io/" },
-  { ctName: "Features1", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features2", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features3", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features4", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features5", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features6", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Legals2", numberOfDocuments: 23, link: "https://prismic.io/" },
-  { ctName: "Features7", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features8", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features9", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features10", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features11", numberOfDocuments: 6, link: "https://prismic.io/" },
-  { ctName: "Features12", numberOfDocuments: 6, link: "https://prismic.io/" },
-];
+  return (hasLocal(ct) ? ct.local.label : ct.remote.label) ?? 'John "CT" Doe';
```

Comment:
😂 damn, shouldn't this function return undefined actually and where it's used we would use the default value ?
Does this default values comes from you or the design ? Because it's quite funny

Reply:
I think returning the ID as the default would make sense :)